enacts/wat_bal/README.md

```diff
@@ -0,0 +1,100 @@
+# ENACTS Maprooms
```

Comment:
agreed but in this case, this is not a copy and paste, at least for the how-to-run the app part. The other 3 apps can only be run through homepage.py whereas this one is disconnected from the multi-pages.

Reply:
Oh, I didn't realize. Why is it disconnected? Is that intended?

In the not-so-long run, we need to have a library of maprooms that you can choose to include/exclude in the config file.

Reply:
I don't know how to add to the multipage but I haven't really tried. I assume it's not going to work because I assume the multipage will fail for cases where we don't have data for the 3 Maprooms (we have so in-house for ZMD only and I am working on Senegal).
But I may very well be utterly wrong.
(also there is no control on the mode for the multipage which is slightly annoying for debugging).

Reply:
OK. Hoping to improve the multi-page setup in the next week or two.